src/OpenTelemetry.AutoInstrumentation/Instrumentations/MessagingAttributes.cs

```diff
@@ -22,12 +25,25 @@ internal static class Kafka
             public const string PartitionOffset = "messaging.kafka.message.offset";
             public const string IsTombstone = "messaging.kafka.message.tombstone";
         }
+
+        internal static class RabbitMq
+        {
+            public const string RoutingKey = "messaging.rabbitmq.destination.routing_key";
+            public const string DeliveryTag = "messaging.rabbitmq.delivery_tag";
+        }
     }
 
     internal static class Values
     {
         public const string KafkaMessagingSystemName = "kafka";
         public const string PublishOperationName = "publish";
         public const string ReceiveOperationName = "receive";
+        public const string ProcessOperationName = "process";
+
+        internal static class RabbitMq
+        {
+            public const string MessagingSystemName = "rabbitmq";
+            public const string DefaultExchangeName = "amq.default";
```

Comment:
According to the semantic conventions, it looks like this should be an empty string when the default exchange is used: https://opentelemetry.io/docs/specs/semconv/messaging/rabbitmq/#rabbitmq-attributes

Reply:
This looks to be consistent with the [RabbitMQ ActivitySource PR](https://github.com/rabbitmq/rabbitmq-dotnet-client/pull/1261) so I suppose this should be fine